client/cypress/integration/query/parameter_spec.js

```diff
@@ -225,11 +283,114 @@ describe('Parameter', () => {
         .click()
         .click(); // workaround for datepicker display bug
 
+      cy.getByTestId('ParameterApplyButton')
+        .click();
+
       cy.get('@Input').then(($input) => {
         const now = Cypress.moment($input.val(), 'DD/MM/YY HH:mm');
         cy.getByTestId('DynamicTable')
           .should('contain', now.format('YYYY-MM-01 HH:mm'));
       });
     });
+
+    it('sets dirty state when edited', () => {
+      expectDirtyStateChange(() => {
+        cy.getByTestId('ParameterName-test-parameter')
+          .find('input')
+          .click();
+
+        cy.get('.ant-calendar-date-panel')
+          .contains('Now')
+          .click();
+      });
+    });
+  });
+
+  describe.only('Apply Changes', () => {
```

Comment:
BTW, don't forget to remove the `.only` before merging ^^
```suggestion
  describe('Apply Changes', () => {
```